docs/api/config/js_kanban_cardshape_config.md

```diff
@@ -151,6 +155,11 @@ To configure the card appearance, in the **cardShape** object you can specify th
         - `id` - (required) a user **ID**
         - `label` - (optional) a user name
         - `avatar` - (optional) a path to the user avatar
+    - `maxCount` - (optional) a maximum count of users displayed on the card
+
+    If you set the `maxCount` property to `true`, you can see only the first assigned user on the card and the number of assigned users near the avatar.
```

Comment:
well, I didn't intend for `true` to work. why would we need `true`?
![Screenshot from 2024-10-15 20-05-14](https://github.com/user-attachments/assets/a9901e19-5fbe-4add-a5f8-966f3d90c55d)
![Screenshot from 2024-10-15 20-05-28](https://github.com/user-attachments/assets/234d9399-7256-41c3-bfc7-f51ce7442769)
![Screenshot from 2024-10-15 20-05-36](https://github.com/user-attachments/assets/4020c459-f5ce-4b6c-8a3c-9f87185e2d3b)
who knows what will be with this setting after review and if it will be in the next release (it is not MERGED and not even VERIFIED, it's just DONE), but for now `true` is not a valid value